sdk/src/main/java/co/decodable/sdk/pipeline/DecodableSecret.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright Decodable, Inc.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package co.decodable.sdk.pipeline;
+
+import co.decodable.sdk.pipeline.internal.DecodableSecretImpl;
+import java.util.Date;
+
+public interface DecodableSecret {
+  String getValue();
```

Comment:
Let's just do `value()`, in line with more modern record naming style rather than 1998 JavaBeans conventions.

Reply:
I assume this is meant to apply to all property accessors?

Reply:
Yes, exactly.